### PR TITLE
Added share option in messageActionSheet.

### PR DESCRIPTION
**Pull request by vishwesh3**
Now one can easily share message on other apps.

ezgif com-gif-maker

### CODE REVIEW

File: src/message/messageActionSheet.js [hunk 1]
```diff
@@ -38,12 +38,19 @@ const muteStream = ({ auth, message, subscriptions }) => {
   muteStreamApi(auth, sub.stream_id);
 };
 
+const shareMessage = ({ message }) => {
+  Share.share({
+    message: message.content.replace(/<(?:.|\n)*?>/gm, '')
```
Comment (borisyankov) [inline src/message/messageActionSheet.js]: Do we want to share the message content or do we want to share a link to it?

**Comment by borisyankov** [comment]
Rebase?

File: src/message/messageActionSheet.js (hunk 1 above)
Comment (vishwesh3) [inline src/message/messageActionSheet.js]: Sharing content will also be useful for those who are not on that realm. (And one can easily get content without clicking link). We can add one more option for link. Slack have "Copy message link."

**Comment by vishwesh3** [comment]
Rebased 👍

**Comment by borisyankov** [comment]
Let's see that version in, we can always improve or change later.

**Pull request merged**